android/app/src/main/java/com/zulipmobile/notifications/NotifyReact.kt

```diff
@@ -76,11 +72,7 @@ val ReactContext.appStatus: ReactAppStatus
         }
     }
 
-internal fun notifyReact(application: ReactApplication, data: Bundle) {
-    // TODO deduplicate this with handleSend in SharingHelper.kt.
-    // Until then, keep in sync when changing.
```

Comment:
We could deduplicate all the way down to `appStatus`, except

Yeah. There's still quite a bit of trickiness here that I'd like to deduplicate: sometimes we set a static/global variable, and sometimes we emit an event, and we need to pick the right one or we're likely to drop or duplicate an event. My ideal version of this and `handleSend` is that they'd each say: here's a `WritableMap`, and here's a little value identifying where the event should go, and then all that logic like the `when (appStatus)` would be in a common function they both invoke.

This change is a helpful step! But let's keep these TODO comments here.